src/containers/views/DataFileEdit.js

```diff
@@ -95,11 +118,16 @@ export class DataFileEdit extends Component {
         className="single">
         {errors.length > 0 && <Errors errors={errors} />}
         <div className="content-header">
-          <Breadcrumbs splat={filename} type="datafiles" />
+          <Breadcrumbs splat={directory || ""} type="data files" />
         </div>
 
         <div className="content-wrapper">
           <div className="content-body">
+            <InputPath
+              onChange={onDataFileChanged}
+              type="data files"
+              path={path} // TODO: Support using `relative_path` from API instead
```

Comment:
For pages and documents, only `filenames` are passed to `InputPath` component. Relative paths are shown in the `Breadcrumb` component. We should be consistent with them. I'm testing locally and seeing `/_data/data_file.yml` instead for InputPath. Before the filename is already shown to users in the Breadcrumbs component. No need to repeat it. Let's abstract that.